External/FEXCore/Source/Interface/Core/JIT/x86_64/ALUOps.cpp

```diff
@@ -417,6 +422,23 @@ DEF_OP(And) {
   mov(Dst, rax);
 }
 
+DEF_OP(Andn) {
+  auto Op = IROp->C<IR::IROp_Andn>();
+  const auto& Lhs = Op->Header.Args[0];
+  const auto& Rhs = Op->Header.Args[1];
+  auto Dst = GRD(Node);
+
```

Comment:
Looks like the CI failure here is likely running in to the case where the destination is also one of the sources.

Reply:
Ah, true. Adjusted it to take that into account